### ISSUE TITLE
[enterprise-4.10] Issue in file service_mesh/v2x/servicemesh-release-notes.adoc

### ISSUE THREAD

**Issue by Senthamilarasu-STA** [openshift/openshift-docs#48896]
### Which section(s) is the issue in?
https://docs.openshift.com/container-platform/4.10/service_mesh/v2x/servicemesh-release-notes.html#component-versions-included-in-red-hat-openshift-service-mesh-version-2-2-1
And all versions below it. 

### What needs fixing?
The component name Jaeger is no longer valid as Service mesh does not use the Upstream directly it used the modified component which is called "Red Hat OpenShift distributed tracing platform" and has to amended accordingly. 
"Jaeger" should be replaced with "Red Hat OpenShift distributed tracing platform(Jaeger)"

Adding Distributed Tracing release notes for ref: https://docs.openshift.com/container-platform/4.10/distr_tracing/distributed-tracing-release-notes.html#component-versions-supported-in-red-hat-openshift-distributed-tracing-version-2-4

**Comment by JStickler** [comment]
This is NOT a bug, while the Product and Operator names have changed, the underlying _component_ name is still Jaeger, the configuration options are still reference Jaeger. and the deployed pods still reference Jaeger.  When I updated the documentation I checked to see where Jaeger was still the correct term to use to make sure that our documentation matched what is displayed in the OpenShift console and in the command line.   The team went through the documentation very carefully when the product name changed to be as accurate as possible with the terminology.
